integrations/timescaledb/timescaledb-metrics/README.md

```diff
@@ -0,0 +1,70 @@
+## Overview
+
+<!-- Sensu Integration description; supports markdown -->
+
+The timescaledb-metrics integration provides a pipeline for delivering sensu metrics to a TimescaleDB table.
+
+This integration includes the following resources:
+
+* `timescaledb-metrics` [pipeline]
+* `timescaledb-metrics` [handler]
+* `timescaledb_dsn` [secret]
+* `sensu/sensu-timescaledb-handler` [asset]
+
+## Dashboards
+
+
+There are no compatible dashboards for this integration.
+
+## Setup
+
+<!-- Sensu Integration setup instructions, including Sensu agent configuration and external component configuration -->
+<!-- EXAMPLE: what configuration (if any) is required in a third-party service to enable monitoring? -->
+
+1. Set up a [TimescaleDB][timescaledb-docs] database with access for your sensu backend.
+
+    See [sensu/sensu-timescaledb-handler][sensu/sensu-timescaledb-handler-github]  docs for sensu-specific schema.
```

Comment:
```suggestion
   See [sensu/sensu-timescaledb-handler][sensu/sensu-timescaledb-handler-github] docs for recommendations.
    
   **Example**: 
    
   ```sql
   CREATE EXTENSION IF NOT EXISTS timescaledb CASCADE;
   CREATE TABLE metrics (
     time    TIMESTAMPTZ         NOT NULL,
     name    TEXT                NOT NULL,
     value   DOUBLE PRECISION    NULL,
     source  TEXT                NOT NULL,
     tags    JSONB
   );
   ```
```